app/redux/reducers/SetsReducer.js

```diff
@@ -93,6 +97,28 @@ const createDefaultState = () => {
     };
 };
 
+// Set default metric
+
+const saveDefaultMetric = (state, action) => {
+    let newWorkoutData = state.workoutData.slice(0);
+    let latestSet = newWorkoutData[newWorkoutData.length - 1];
+    let setIndex = newWorkoutData.findIndex( set => set.setID === action.setID );
+    let set = newWorkoutData[setIndex];
+
+    let changes = {};
+    
+    // Check if set is empty before allowing metric to change
+    if (!set.exercise && !set.weight && !set.rpe && !set.tags && !set.reps && !set.tags && !set.videoFileURL) {
```

Comment:
do tags only once

tags and reps should check length AND not empty state

Reply:
Done